test/built-ins/Temporal/PlainDateTime/prototype/until/round-relative-to-receiver.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.until
+description: Rounding happens relative to receiver
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+const dt1 = new Temporal.PlainDateTime(2019, 1, 1);
+const dt2 = new Temporal.PlainDateTime(2020, 7, 2);
+const options = { smallestUnit: "years", roundingMode: "halfExpand" };
+
+TemporalHelpers.assertDuration(
+  dt1.until(dt2, options),
+  2, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+  "rounds relative to the receiver (1)"
```

Comment:
I'd recommend using "(positive case)" and "(negative case)" like you did in the corresponding `since()` test.

Reply:
done